### PR TITLE
Update code block tab designs and add option to display static framework logos in group

### PR DESCRIPTION
**Pull request by robbieaverill**
Resolves https://github.com/chec/ui-library/issues/372

### CODE REVIEW

File: src/assets/svgs/frameworks/angular.svg [hunk 1]
```diff
@@ -0,0 +1,4 @@
+<svg width="32" height="32" viewBox="0 0 32 32" fill="none" xmlns="http://www.w3.org/2000/svg">
+    <path d="M13.3761 16.864H16.0961H18.8161L16.0961 10.32L13.3761 16.864Z" fill="#D6E4F7"/>
```
Comment (kvisca) [inline src/assets/svgs/frameworks/angular.svg:2]: Fill should be ` fill="currentColor"` so we can control the color more easily. This applies to all the SVGs added

**Review by kvisca — CHANGES_REQUESTED** [review]
Besides that one change, LGTM!

File: src/assets/svgs/frameworks/angular.svg (hunk 1 above)
Comment (robbieaverill) [reply src/assets/svgs/frameworks/angular.svg:2]: Where do we define `currentColor`?

Comment (robbieaverill) [reply src/assets/svgs/frameworks/angular.svg:2]: It's a convention thing, I see. The Vue logo has two colours in it, how do you suggest I handle that? 
image

Comment (kvisca) [reply src/assets/svgs/frameworks/angular.svg:2]: The fill would be on the <path> (end of line 2 in that example).  The two-tone one is a bit of a challenge. I suppose we could do a `fill="currentcolor"` and  a `fill-opacity="0.4"` and try to match it as close as we can.

Comment (robbieaverill) [reply src/assets/svgs/frameworks/angular.svg:2]: How would you feel about me making the second line currentColor and leaving the other one the way it is? There'll be a phase 2 of this PR at some point in the future to make the logos configurable

Comment (robbieaverill) [reply src/assets/svgs/frameworks/angular.svg:2]: Actually your suggestion works perfectly, I'm going to use that

**Review by kvisca — APPROVED** [review]
Awesome LGTM!

**Pull request merged**